### PR TITLE
Fix UI tabs inconsistency

### LINKED ISSUE #5718: Unify Inconsistent UI elements - Tabs
**Describe the bug**

Unify Inconsistent UI elements

**To Reproduce**
View tabs on Profile, Quests, Grants

**Expected behavior**

Look and behave the same. 
Make Profile and Quests match Grants.

**Screenshots**

**Grants**
Screen Shot 2020-01-07 at 3 49 56 AM
**Quests**
Screen Shot 2020-01-07 at 3 29 11 AM
**Profiles**
Screen Shot 2020-01-07 at 4 04 36 AM

**Desktop (please complete the following information):**

- OS:MAC
- Browser: Chrome
- Browser Version:

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description
Fix UI inconsistency on tabs.

##### Refers/Fixes
Fixes #5718 

##### Testing
Checkout profile page tabs and quests tabs to view it.

### CODE REVIEW

**Comment by iamonuwa** [comment]
image
For profile

I was unable to load quests locally, but fixed it nonetheless. It works as well

**Comment by danlipert** [comment]
@iamonuwa Can you post some more screenshots or video showing the changes? thanks!

**Comment by iamonuwa** [comment]
> @iamonuwa Can you post some more screenshots or video showing the changes? thanks!

Sure

**Comment by iamonuwa** [comment]
screen-capture
@danlipert, this is for the profile. I don't have quests loaded locally to test with. But the quests changes are same and will work

File: app/dashboard/templates/profiles/tabs.html [hunk 1]
```diff
@@ -2,112 +2,93 @@
...
         {% if show_resume_tab %}
-          <li class="nav-item">
-            <a class="nav-link nav-line  {% if tab == 'resume' %}active{%endif%}" href="{{profile.url}}/resume">
-              {% if not profile.show_job_status and is_my_profile %}
-                <i class="fas fa-eye"></i>
-              {% endif %}
-                Resume
-            </a>
-          </li>
+        <button type="button" id="nav-description" href="{{profile.url}}/resume" class="text-center section-tab {% if tab == "resume" %} active {% endif %}">
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tabs.html]: indent this by 1

File: app/dashboard/templates/profiles/tabs.html [hunk 2]
```diff
@@ -2,112 +2,93 @@
...
         {% if all_feedbacks|length %}
-          <li class="nav-item">
-            <a class="nav-link nav-line  {% if tab == 'ratings' %}active{%endif%}" href="{{profile.url}}/ratings">
-              Ratings <span>{{all_feedbacks|length}}</span>
-            </a>
-          </li>
+        <button type="button" id="nav-description" href="{{profile.url}}/ratings" class="text-center section-tab {% if tab == "ratings" %} active {% endif %}">
+          {% trans "RATINGS" %}
+          <span  class="nav-badge">({{all_feedbacks|length}})</span>
+        </button>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tabs.html]: indent

File: app/dashboard/templates/profiles/tabs.html [hunk 3]
```diff
@@ -2,112 +2,93 @@
...
         {% if active_bounties_count %}
-          <li class="nav-item ">
-            <a class="nav-link nav-line  {% if tab == 'active' %}active{%endif%}" href="{{profile.url}}/active">
-              Active Issues <span>{{active_bounties_count}}</span>
-            </a>
-          </li>
+        <button type="button" id="nav-description" href="{{profile.url}}/active" class="text-center section-tab {% if tab == "active" %} active {% endif %}">
+          {% trans "ACTIVE ISSUES" %}
+          <span  class="nav-badge">({{active_bounties_count}})</span>
+        </button>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tabs.html]: indent

File: app/dashboard/templates/profiles/tabs.html [hunk 4]
```diff
@@ -2,112 +2,93 @@
...
         {% if portfolio_count or is_my_profile %}
-          <li class="nav-item">
-            <a class="nav-link nav-line  {% if tab == 'portfolio' %}active{%endif%}" href="{{profile.url}}/portfolio">
-              Portfolio <span>{{portfolio_count}}</span>
-            </a>
-          </li>
+        <button type="button" id="nav-description" href="{{profile.url}}/portfolio" class="text-center section-tab {% if tab == "portfolio" %} active {% endif %}">
+          {% trans "PORTFOLIO" %}
+          <span  class="nav-badge">({{portfolio_count}})</span>
+        </button>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tabs.html]: indent

File: app/dashboard/templates/profiles/tabs.html [hunk 5]
```diff
@@ -2,112 +2,93 @@
...
         {% if profile.organizations|length %}
-          <li class="nav-item">
-            <a class="nav-link nav-line  {% if tab == 'orgs' %}active{%endif%}" href="{{profile.url}}/orgs">
-              Orgs <span>{{profile.organizations|length}}</span>
-            </a>
-          </li>
+        <button type="button" id="nav-description" href="{{profile.url}}/orgs" class="text-center section-tab {% if tab == "orgs" %} active {% endif %}">
+          {% trans "ORGS" %}
+          <span  class="nav-badge">({{profile.organizations|length}})</span>
+        </button>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tabs.html]: indent

**Comment by danlipert** [comment]
> screen-capture
> @danlipert, this is for the profile. I don't have quests loaded locally to test with. But the quests changes are same and will work

Just make some quests in your local - its not difficult

**Comment by iamonuwa** [comment]
> > screen-capture
> > @danlipert, this is for the profile. I don't have quests loaded locally to test with. But the quests changes are same and will work
> 
> Just make some quests in your local - its not difficult

Will do that and send in the video

**Comment by thelostone-mc** [comment]
@iamonuwa while you're at it -> could you resolve the conflicts too ?

**Comment by iamonuwa** [comment]
> @iamonuwa while you're at it -> could you resolve the conflicts too ?

Aye captain 😁✊

**Comment by iamonuwa** [comment]
Closing this in favor of #5759

**Pull request closed without merging**